src/__deprecated__/components/form/form.component.js

```diff
@@ -355,6 +356,10 @@ class FormWithoutValidations extends React.Component {
     };
   }
 
+  isHTMLElement(child) {
+    return isElement(child) && typeof child.type === 'string';
```

Comment:
What's the purpose of this condition?
```js
typeof child.type === 'string'
```
Maybe it'd be good to add a comment to the code to clarify its purpose?

Reply:
Thanks for adding that comment in 6342174 :thumbsup: